test/sharness/t0250-files-api.sh

```diff
@@ -460,6 +460,24 @@ test_files_api() {
 		ipfs files rm -r /test_dir &&
 		ipfs files rm -r /test_file
 	'
+
+	test_expect_success "make a directory and a file" '
+		ipfs files mkdir /adir &&
+		echo "blah" | ipfs files write --create /foobar
+	'
+
+	test_expect_success "copy a file into a directory" '
+		ipfs files cp /foobar /adir/
+	'
```

Comment:
This test case could've been created from the get go. Let's test more thoroughly from the beginning.